lib/features/tabs/screens/tab_screen.dart

```diff
@@ -1,56 +1,89 @@
 import 'package:elvan/features/cart/screens/cart_screen.dart';
 import 'package:elvan/features/favorite/screens/favorite_screen.dart';
 import 'package:elvan/features/home/screens/home_screen.dart';
+import 'package:elvan/features/login/screens/login_screen.dart';
 import 'package:elvan/shared/components/constants/app_colors.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
 
-class TabScreen extends StatefulWidget {
-  const TabScreen({super.key});
-
-  @override
-  State<TabScreen> createState() => _TabScreenState();
-}
-
-class _TabScreenState extends State<TabScreen> {
-  int _selectedIndex = 0;
-
-  void _onItemTapped(int index) {
-    setState(() {
-      _selectedIndex = index;
-    });
-  }
-
+class TabScreen extends HookWidget {
   static const List<Widget> _pages = [
     HomeScreen(),
     FavoriteScreen(),
     CartScreen(),
+    LoginScreen(),
```

Comment:
Profile screen will be here. When user is not logged in they will be redirected to Login. Will explain to you later.